src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Handles Duke's File I/O.
+ */
+public class Ui {
+    private final Scanner sc = new Scanner(System.in);
+    private final String line = "------------------------------"
+            + "------------------------------";
+    private final String logo = " ____        _        \n"
```

Comment:
The `logo` and `line` variable names should be in UPPER_SNAKE_CASE to conform with the style guide, since they are being used as constants. (I think you may also want to make them static, since they are shared among all instances of this object).